### PR TITLE
release 8.13.0

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: news/coq-8-13-0-is-out [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Highlights:
+<ul>
+<li>Introduction of primitive persistent arrays in the core language, implemented using imperative persistent arrays. </li>
+<li>Introduction of definitional proof irrelevance for the equality type defined in the SProp sort. </li>
+<li>Many improvements to the handling of notations, including number notations, recursive notations and notations with bindings. A new algorithm chooses the most precise notation available to print an expression, which might introduce changes in printing behavior.</li>
+</ul>
+</p>
+
+
+<p>Please see <a href="https://coq.github.io/doc/v8.13/refman/changes.html#version-8-13" rel="nofollow">the changelog</a> to learn more about this release.</p>
```
Comment (Zimmi48) [inline news/coq-8-13-0-is-out:22]: I guess you copied the HTML code from GitHub, but I would remove the `rel="nofollow"` from here :wink:

**Review by Zimmi48 — APPROVED** [review]
LGTM

File: news/coq-8-13-0-is-out (hunk 1 above)
Comment (gares) [reply news/coq-8-13-0-is-out:22]: I took it from the release of 8.12

Comment (Zimmi48) [reply news/coq-8-13-0-is-out:22]: Ah, my bad then :wink:

**Pull request merged**